src/components/actionMenu/actionMenu.tsx

```diff
@@ -19,14 +26,16 @@ type Action<T> = {
 interface ActionMenuProps<T = unknown> {
   item: T;
   actions: Action<T>[];
+  popperProps?: Partial<PopperProps>;
```

Comment:
@cdcabrera David, if answering this won't take too much of your time, what would be the ideal fix for this use case ? You can just point me to the right component and I can take a look. Thank you.